lib/index.js

```diff
@@ -43,12 +43,11 @@ module.exports = function Conversation(bot) {
     };
 
     /**
-     * Returns an existing Dialog with a given user id.
+     * Returns an existing Dialog with a given user or room id.
      * @param userId
      * @returns {*}
      */
-    this.talkingTo = function talkingTo(userId) {
-        return _talkingTo[userId];
+    this.talkingTo = function talkingTo(id) {
+        return _talkingTo[id];
     };
 };
-
```

Comment:
In this file I'd refactor `(type === 'user')? msg.message.user.id: msg.message.room,` into a private function `getId(msg)` so you don't have to repeat the same code block multiple times.